Sources/Publish/Internal/PodcastFeedGenerator.swift

```diff
@@ -107,7 +110,7 @@ private extension PodcastFeedGenerator {
                     .summary(item.description),
                     .explicit(metadata?.isExplicit ?? false),
                     .duration(audioDuration),
-                    .image(config.imageURL),
+                    .image(imageURL ?? config.imageURL),
```

Comment:
Wouldn't it be better to exclude the `image` tag altogether when the item has no specific image?

As far as I've been able to tell, all clients will fall back to the main `<itunes:image>` at the channel level when the episode has no custom artwork. For shows with lots of episodes, having these duplicate tags in the feed add up and make the feed larger than it needs to be.